NitroxClient/GameLogic/HUD/Components/uGUI_PlayerPingEntry.cs

```diff
@@ -0,0 +1,254 @@
+            Modal.Get<ConfirmModal>()?.Show($"{text}?", () =>
+            {
+                packetSender.Send(new ServerCommand($"warp {player.PlayerName} {localPlayer.PlayerName}"));
+            });
+        });
+
+        RefreshButtonsVisibility();
+    }
+
+    public void UpdateButtonsPosition()
+    {
+        float offset = 540f;
+
+        MuteObject.transform.localPosition = new(
+            0f + offset,
+            MuteObject.transform.localPosition.y,
+            MuteObject.transform.localPosition.z);
+        KickObject.transform.localPosition = new(
+            80f + offset,
+            KickObject.transform.localPosition.y,
+            KickObject.transform.localPosition.z);
+        TeleportToObject.transform.localPosition = new(
+            160f + offset,
+            TeleportToObject.transform.localPosition.y,
+            TeleportToObject.transform.localPosition.z);
+        TeleportToMeObject.transform.localPosition = new(
+            240f + offset,
+            TeleportToMeObject.transform.localPosition.y,
+            TeleportToMeObject.transform.localPosition.z);
+    }
+
+    private void ClearButtonListeners()
+    {
+        GetToggle(MuteObject).onValueChanged.RemoveAllListeners();
+        GetToggle(KickObject).onValueChanged.RemoveAllListeners();
+        GetToggle(TeleportToObject).onValueChanged.RemoveAllListeners();
+        GetToggle(TeleportToMeObject).onValueChanged.RemoveAllListeners();
+    }
+
+    public void AssignSprites()
+    {
+        // NB: Those textures MUST be exported with a Texture Type of "Sprite (2D and UI)", else they will look blurry not matter what
+        // NB 2: Those textures for the buttons are scaled 68x61 but the image inside but not hit the borders to have a better render
+        MutedSprite = parent.GetSprite("muted@3x");
+        UnmutedSprite = parent.GetSprite("unmuted@3x");
+        KickSprite = parent.GetSprite("kick@3x");
+        TeleportToSprite = parent.GetSprite("teleport_to@3x");
+        TeleportToMeSprite = parent.GetSprite("teleport_to_me@3x");
+        
+        MuteObject.FindChild("Eye").GetComponent<Image>().sprite = UnmutedSprite;
+        KickObject.FindChild("Eye").GetComponent<Image>().sprite = KickSprite;
+        TeleportToObject.FindChild("Eye").GetComponent<Image>().sprite = TeleportToSprite;
+        TeleportToMeObject.FindChild("Eye").GetComponent<Image>().sprite = TeleportToMeSprite;
+    }
+
+    private void RefreshMuteButton(bool muted)
+    {
+        GetToggle(MuteObject).SetIsOnWithoutNotify(muted);
+        GetTooltip(MuteObject).TooltipText = Language.main.Get(muted ? "Nitrox_Unmute" : "Nitrox_Mute");
+        MuteObject.FindChild("Eye").GetComponent<Image>().sprite = muted ? MutedSprite : UnmutedSprite;
+    }
+
+    private void RefreshButtonsVisibility()
+    {
+        LocalPlayer localPlayer = NitroxServiceLocator.LocateService<LocalPlayer>();
+        
+        bool isNotLocalPlayer = !IsLocalPlayer || true;
```

Comment:
```suggestion
        bool isNotLocalPlayer = !IsLocalPlayer;
```
Forgot to remove that testing part (useful because you can test with only one client)

Reply:
Just letting this comment there for testing purposes (when alone) but it's no longer in the code